### DISCUSSION TITLE
We've migrated to nuxt3 in the last year and deployed to prod today 🎉

### DISCUSSION THREAD

**Discussion by toooni** [nuxt/nuxt#16297] [General]
We have a quite “big” and somewhat complicated page https://www.weekend4two.com (unfortunately only available in german, and french). Until today, we've used PHP (Symfony) to render html/vue2 with the help of twig.

We started planning a complete overwrite for our frontend a year ago and are quite happy to announce that we have migrated to nuxt3, today. During this time, we've also rewrote our current design almost completely with taildwindcss, which showed to be a great match with vite and nuxt3. On the backend side, we are still using symfony as our API.

Thanks to the nuxt3 team for this awesome piece of software 🎉 ❤️

**Comment by Diizzayy** [comment]
@toooni Congratulations on the launch! The website looks great.

Certainly happy to hear that you've enjoyed working with Nuxt.

**Comment by TheAlexLichter** [comment]
Good job and congrats to the launch! 🔥
